test/application_system_test_case.rb

```diff
@@ -0,0 +1,5 @@
+require "test_helper"
```

Comment:
You switched back from RSpec?

Reply:
This is something that was autogenerated. I did not make any change

Reply:
You can skip the default testing framework when generating a new project with `rails new -T`; then you can go with RSpec.